spec/support/self_service/firms_index_page.rb

```diff
@@ -1,13 +1,15 @@
 module SelfService
   class FirmsIndexPage < SitePrism::Page
     set_url '/self_service/firms'
-    set_url_matcher %r{/self_service/firms}
+    set_url_matcher %r{/self_service}
```

Comment:
Should `/self_service` be a redirect to `/self_service/firms` now then? Rather than having a duplicate endpoint for the same result?